astropy/io/misc/hdf5.py

```diff
@@ -226,7 +227,7 @@ def write_table_hdf5(
     ----------
     table : `~astropy.table.Table`
         Data table that is to be written to file.
-    output : str or :class:`h5py.File` or :class:`h5py.Group`
+    output : str or os.PathLike[str] or :class:`h5py.File` or :class:`h5py.Group`
```

Comment:
Needs a change log under feature?

Reply:
sorry I missed this for so long. It's done now !